src/briefcase/platforms/linux/__init__.py

```diff
@@ -1,5 +1,260 @@
-DEFAULT_OUTPUT_FORMAT = "appimage"
+import subprocess
+import sys
+from pathlib import Path
+from typing import List
+
+from briefcase.commands.create import _is_local_requirement
+from briefcase.commands.open import OpenCommand
+from briefcase.config import AppConfig
+from briefcase.exceptions import BriefcaseCommandError
+
+DEFAULT_OUTPUT_FORMAT = "system"
+
+DEBIAN = "debian"
+REDHAT = "redhat"
+ARCH = "archlinux"
 
 
 class LinuxMixin:
     platform = "linux"
+
+    def _release_details(self, filename):
+        """Read an /etc/X-release file.
+
+        :param filename: The name of a file in the `/etc` folder
+        :returns: The string content of the release file, or None if the file
+            does not exist.
+        """
+        path = Path("/etc") / filename
+        if path.exists():
+            with path.open(encoding="utf-8") as f:
+                return f.read()
+        return None
+
+    def host_distribution(self):
+        """Identify the local Linux distribution and version.
+
+        :returns: A tuple of (vendor, version).
+        """
+        # Try to identify the local Linux system
+        # * Arch provides `/etc/arch-release`.
+        # * Manjaro is an Arch derivative, and puts identifying content in
+        #   `/etc/arch-release`
+        # * Redhat-derived distros all provide `/etc/redhat-release`
+        # * Fedora also provides `/etc/fedora-release`, which contains the
+        #   version number
+        # * Almalinux also provides `/etc/almalinux-release`, which contains the
+        #   version number
+        # * Debian-based distros use `lsb_release`; this *may* be present on
+        #   other distros, but usage of code names is wierd (e.g., Fedora 34 has
+        #   a code name "thirtyfour", but the docker image isn't tagged with
+        #   that code name)
```

Comment:
Note that modern Linux systems have a [standardized](https://www.freedesktop.org/software/systemd/man/os-release.html), structured `/etc/os-release` file. Python 3.10 [can read it](https://docs.python.org/3/library/platform.html#platform.freedesktop_os_release) and vendoring that code for older versions shouldn't be hard.

Reply:
Ah - I wasn't aware of the existence of `/etc/os-release`. Thanks - that will simplify things considerably!